### ISSUE TITLE
import-ipynb 0.1.4 doesn't work with Python 3.12.0

### ISSUE THREAD

**Issue by ampilnick** [axil/import-ipynb#5]
Hello, dear Lev!
Thank you for such a useful tool – _**import-ipynb**_ ( https://pypi.org/project/import-ipynb/ )!
Unfortunately, it doesn’t work with Python 3.12.0 (see fig. cells 1 & 3).
![unnamed](https://github.com/axil/import-ipynb/assets/37598012/e867e49d-728c-4698-bf2a-f3b7e7b61e6a)
(Sys: Win11, VSCode. Cell_2: the proofs of the file existence)

I kindly ask you to fix it, please!

With best regards,
yours
Nikolay

**Comment by axil** [comment]
Hi, @ampilnick! I've looked into the issue. It looks feasible to fix, just a little bit tricky. Will hopefully be done soon.

**Comment by mvanec** [comment]
Curious if this has been looked at at all. It still does not work in Python 3.12.x.

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[5], [line 1](vscode-notebook-cell:?execution_count=5&line=1)
----> [1](vscode-notebook-cell:?execution_count=5&line=1) from Triangle import Triangle
      [2](vscode-notebook-cell:?execution_count=5&line=2) from Triangle import TrianglePlot

ModuleNotFoundError: No module named 'Triangle'
```

**Comment by axil** [comment]
@mvanec Fixed in version 0.2
